recipes/dialign2/patchdialign.txt

```diff
@@ -1,5 +1,5 @@
---- /homes/biertruck/fall/Downloads/dialign_package/src/dialign.c	2005-02-21 17:32:55.000000000 +0100
-+++ /homes/biertruck/fall/Downloads/dialign_package/src/fixed_dialign.c	2018-07-26 16:46:41.962909610 +0200
```

Comment:
should this file now be removed?

Reply:
yes, the patch is now in the subdir patches